starter/source/constraints/general/rbe3/hm_read_rbe3.F

```diff
@@ -444,12 +444,10 @@ SUBROUTINE INIRBE3(IRBE3  ,LRBE3  ,FRBE3  ,SKEW   ,X     ,
      .                   R1=WMIN)
            END IF
         END IF
-C------ remove JR of NS if no dof of rotation		
-		IF (IRODDL==0 .OR.IN(NS)==ZERO) THEN
-		 IC = IRBE3(4,I)
-		 ICT = IC/512
-		 IRBE3(4,I) = ICT*512
-		END IF
+		IC = IRBE3(4,I)
```

Comment:
Hi @qiangRadioss 

could you reindent correctly ( remove the tab )

Thanks
Sébastien

Reply:
done, thanks.